### ISSUE TITLE
How do I force https?

### ISSUE THREAD

**Issue by thedonquixotic** [nextcloud/nextcloud-snap#647]
Hi. 

About a month ago I installed NextCloud on a new desktop that I'm turning into a personal cloud server. It extremely easy. I did some Desktop Environment hopping, screwed up my installation and decided to go back to Gnome on Ubuntu 18.04. After a couple of weeks learning more stuff I decided to reinstall NextCloud. 

But I can't get the webinstaller to work the way I remembered, so I thought.... maybe I didn't use the webinstaller, Maybe I just used a snap package. I have installed NextCloud with the Snap package but I can't figure out how to force https on the server. I thought this was a command line thing while installing, at least according to my memory though that is pretty hazy at this point.

**Comment by kyrofa** [comment]
Check out `nextcloud.enable-https --help`. Does that look familiar?

**Comment by thedonquixotic** [comment]
Unfortunately that is getting me an error code:

```
david@TheWiz:~$ sudo nextcloud.enable-https --help
[sudo] password for david: 
Invalid option: --
```

**Comment by thedonquixotic** [comment]
I also can't find the physical location of my files in Nautilus. I did some searching on it and looked in the var folder, and that did not work. I think I installed it from snap wrong. 

This definitely doesn't ring a lot of bells though. For one the snap creates localhost.localdomain and I remember it being my IP address. And also the NextCloud folder was just a folder called NextCoud in my $Home folder. So.... I think I did something else. The web installer maybe? Is that even possible to do locally? Because when I try just opening up the php file in my browser it doesn't work... which make sense. I'm a designer learning how to code, so I'm still learning a lot of but usually I have to have my MAMP stack running in order to access WordPress Admin. But I'm not really clear on how that should be done here, with the web installer or whatever. I certainly don't remember downloading mamp on my previous linux install though. And I've searched a bunch of varations and checked my web history and I can't find whatever original tutorial I followed the first time.

**Comment by kyrofa** [comment]
Yikes, sorry, I meant `-h`.

**Comment by thedonquixotic** [comment]
Thanks. Yes, that worked. And no it does not look familiar. I am now sure that I didn't actually get https set up. I just got it running on my local network with http. That said, I can't find the location of the actual physical files and searching online I haven't found it like people have suggested, which makes me think that I didn't install it with snap, I did install it with the webstarter somehow, I don't know how I did but I've put a post on the support forum asking for help, and perhaps someone will be able to give me a suggestion.

Thank you that was really helpful Kyrofa. 

I'm going to close this issue as it's been resolved. As for what to do next, I guess I'll just have to see what people on the forum say.
